README.md

```diff
@@ -274,6 +274,12 @@ Auth0
     }
 ```
 
+### Custom Domains
+
+If you are using the [Custom Domains](https://auth0.com/docs/custom-domains) feature, it's important you do not use the custom domain value when configuring the `.audience`. Please use the Auth0 domain specified for your Application in the [Auth0 Dashboard](https://auth0.com/docs/dashboard). 
```

Comment:
I find it confusing. After checking the [docs](https://auth0.com/docs/custom-domains#-service-not-found-) (which you can refer) I suggest:
  
If you are using the [Custom Domains](https://auth0.com/docs/custom-domains) feature and require to request the `/userinfo` or `/api/v2/` (I'm not 100% sure about this one) audience, please use the Auth0 domain specified for your Application in the dashboard.

Example:  `.audience("https://{YOUR_AUTH0_DOMAIN}/userinfo")`

Reply:
good point, I was only thinking about Auth0 audiences.